docs/dev/tools/common-errors.md

```diff
@@ -184,27 +184,25 @@ This can happen when there is a slow internet connection, when the module is try
 **Solution:**
 
 - Try using a faster internet connection.
-- If the problem persists, try increasing the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables in the module's JSON configuration file.
-  For example:
-
-  ```json {class="line-numbers linkable-line-numbers"}
-  {
-    "modules": [
-      {
-        ...
-        "env": {
-          "VIAM_MODULE_STARTUP_TIMEOUT": "6m30s",
-          "VIAM_RESOURCE_CONFIGURATION_TIMEOUT": "3m0s"
-        }
-      }
-    ]
-  }
-  ```
-
 - If you are the module author, consider packaging the module with required dependencies so they don't need to be downloaded on startup.
   For Python modules, you can package your module with dependencies by using the PyInstaller steps when [uploading your module](/operate/get-started/other-hardware/#upload-your-module).
+- If the problem persists, try setting the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables on your machine.
+  To configure these variables on a Linux machine that uses the `bash` shell:
+
+  1. Connect to your machine using SSH.
+  1. Open your user account's <file>~/.bashrc</file> file.
+  1. Add the following lines to the end of the file:
+
+     ```bash
+     export VIAM_MODULE_STARTUP_TIMEOUT="6m30s"
+     export VIAM_RESOURCE_CONFIGURATION_TIMEOUT="3m0s"
+     ```
+
+  1. Run `source ~/.bashrc` to load your `bash` configuration changes.
+  1. Finally, run `sudo systemctl restart viam-server` to restart `viam-server`.
+ 
```

Comment:
```suggestion
For a value, pass sequence of numbers and a units, e.g. "6m30s", "6.5m", or "630000ms".
You can specify units in "ns" (nanoseconds), "us"/"µs" (microseconds), "ms" (milliseconds), "s" (seconds), "m" (minutes), and "h" (hours). 
```
Implementing a suggestion: removing the explicit instructions entirely. Providing a little more information about the units that viam-server can parse for env var time duration input. ns & us & h are probably overkill, but they don't hurt.